Code/Framework/AzToolsFramework/AzToolsFramework/aztoolsframework_files.cmake

```diff
@@ -407,8 +407,6 @@ set(FILES
     UI/DocumentPropertyEditor/DPEComponentAdapter.cpp
     UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp
     UI/DocumentPropertyEditor/DocumentPropertyEditor.h
-    UI/DocumentPropertyEditor/DocumentPropertyEditorSettings.cpp
-    UI/DocumentPropertyEditor/DocumentPropertyEditorSettings.h
```

Comment:
Shouldn't the `DocumentPropertyEditorSettings.h` also be removed as well?

Reply:
I'm confused, it is, right? It's in the line right where you added this comment